tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -439,16 +439,9 @@ struct LaunchBatchMatMul<GPUDevice, Scalar> {
         bool blas_launch_status =
             stream
                 ->ThenBlasGemm(blas_transpose_b, blas_transpose_a, n, m, k,
-<<<<<<< HEAD
-                               static_cast<Coefficient>(alpha), *(b_ptrs[0]),
-                               adj_y || trans_y ? k : n, *(a_ptrs[0]),
-                               adj_x || trans_x ? m : k,
-                               static_cast<Coefficient>(beta), c_ptrs[0], n)
-=======
                                *(b_ptrs[0]), adj_y || trans_y ? k : n,
                                *(a_ptrs[0]), adj_x || trans_x ? m : k,
                                c_ptrs[0], n)
->>>>>>> google_upstream/master
                 .ok();
```

Comment:
@deven-amd, @ekuznetsov139
I think this is the location I made the mistake to accept the upstream changes instead of ours. By accepting their changes, we call the simplified API call (assuming alpha=1.0 and beta=0.0). I will revert this part back.

Reply:
correct.

Reply:
basically this upstream commit ( https://github.com/tensorflow/tensorflow/commit/0908c2f2397c099338b901b067f6495a5b96760b ), amongst many other changes,  allows `alpha` / `beta` to have a different datatype than that of the inputs, and also creates a version in which `alpha` / `beta` are hardcoded to 1/0

we need to call the version which takes in the `alpha` / `beta` as args